requirements.txt

```diff
@@ -29,6 +29,7 @@ dataclasses-json~=0.5.7
 etcetra==0.1.15
 faker~=13.12.0
 graphene~=2.1.9
+graypy==2.1.0
```

Comment:
I'd like to mark it as an optional, extra dependency, following [PEP-508](https://peps.python.org/pep-0508/#extras), so that it is installed only when explicitly requested like `pip install backend.ai-manager[graylog]`.

Reply:
I set this as an extra so that we can install it only if specified as you mentioned.